metricflow/mf_logging/pretty_print.py

```diff
@@ -424,21 +424,32 @@ def mf_pformat_many(  # type: ignore
     include_object_field_names: bool = True,
     include_none_object_fields: bool = False,
     include_empty_object_fields: bool = False,
+    raw_strings: bool = False,
```

Comment:
nit: can we rename this to `preserve_raw_strings` or something? The `raw_strings` name isn't descriptive at the call site, and since the parameter value is a bool we're just requiring new devs to go read the docstring or function body to find out what `raw_strings` does.

Reply:
Good point - updated.